x-pack/plugins/fleet/public/applications/fleet/sections/agents/components/agent_upgrade_modal/constants.tsx

```diff
@@ -5,8 +5,9 @@
  * 2.0.
  */
 
-// Available versions for the upgrade of the Elastic Agent
+export const MINIMUM_SUPPORTED_VERSION = '8.3.0';
```

Comment:
nit: could be a little more descriptive
```suggestion
export const ROLLING_UPGRADE_MINIMUM_SUPPORTED_VERSION = '8.3.0';
```